pkgs/by-name/op/opcua-commander/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  typescript,
+  esbuild,
+  makeWrapper,
+  nodejs,
+}:
+buildNpmPackage rec {
+  pname = "opcua-commander";
+  version = "0.37.0";
+
+  src = fetchFromGitHub {
+    owner = "node-opcua";
+    repo = pname;
+    rev = version;
+    hash = "sha256-WoRbZTgnkWqtHDgguQns7A4pOKLN+m6n4hZ+imv8nWg=";
+  };
+
+  npmDepsHash = "sha256-4rZZESyRSDsmMoVVQJbQnePBn+hwXw0w1BKN2rcRTvs=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/jgi8br3zwqj3dnqdabrs9z4mws78ss2r-opcua-commander-0.37.0-npm-deps.drv':
         specified: sha256-4rZZESyRSDsmMoVVQJbQnePBn+hwXw0w1BKN2rcRTvs=
            got:    sha256-Ux1X/3sam9WHrTfqoWv1r9p3pJOs6BaeFsxHizAvjXA=
```

Reply:
not really sure how these two got the incorrect values... I must have ended up pushing one of the previous versions, sorry for that, it should be fixed now.